<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowCadastrar</class>
 <widget class="QMainWindow" name="WindowCadastrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 80);
color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
	border-radius:3px;
	color: rgb(54, 54, 81);
	background-color: rgb(223, 223, 223);
}

QPushButton:hover{
	
	
	color: rgb(54, 54, 81);
	background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>280</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 8pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;                    N O C z  @ All CopyRight   - 2019&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>231</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>231</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius:3px;
	color:rgb(23,25,25);
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover{
background-color:rgb(170,170,255);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblCadNo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>225</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0,0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt; Aten√ßao erro ao cadastrar o usuario !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblNome">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>231</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>231</width>
      <height>22</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>231</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    
	color: rgb(56, 56, 83);
	border-radius:3px;
	color:rgb(23,25,25);
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover{
background-color:rgb(170,170,255);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>20</y>
      <width>421</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Shonar Bangla</family>
      <pointsize>22</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Shonar Bangla&quot;;
color: rgb(164, 211, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Cadastrar &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Nome :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCadOk">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>421</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(112, 193, 255);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Usuario cadastrado com sucesso!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrar_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
	border-radius:3px;
	color: rgb(54, 54, 81);
	background-color: rgb(223, 223, 223);
}

QPushButton:hover{
	
	
	color: rgb(54, 54, 81);
	background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <zorder>btnCadastrar_2</zorder>
   <zorder>btnCadastrar</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lblEmail</zorder>
   <zorder>lblCadNo</zorder>
   <zorder>lblNome</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>lblCadOk</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
